### ISSUE TITLE
Assertion failed when running `flow type-at-pos`

### ISSUE THREAD

**Issue by frantic** [facebook/flow#691]
Input file: [a.js](https://gist.github.com/frantic/9b4b8325fc967055e91e)
```
$ flow --version
Flow, a static type checker for JavaScript, version 0.14.0
$ flow type-at-pos ./a.js 23 15
File "/private/tmp/frantic-lAdtia/a.js", line 23, characters 15-15:
File "src/typing/flow_js.ml", line 664, characters 9-15: Assertion failed
```

src/typing/flow_js.ml, line 664: `| _ -> assert false (** TODO **)`

Looks like this happens when the type of the variable I'm trying to inspect has a custom type (defined with `type x = ...`).

**Comment by mroch** [comment]
this is sorta-fixed in the upcoming 0.15. it will say `(unknown)` instead of crashing. tracking fixing the `(unknown)` in #463

**Comment by mroch** [comment]
^ fixed the `(unknown)` issue
